styles/style.css

```diff
@@ -0,0 +1,131 @@
+body {
+  background-color: #fff;
+}
+
+.device-container {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.device-look {
+  width: 375px;
+  height: 667px;
+  border-radius: 10px;
+  border-style: solid;
+  border-width: 5px 5px 50px 5px;
+  overflow-y: scroll;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo-container {
+  width: 114px;
+  height: 48px;
+  margin-left: 12px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+}
+
+.logo {
+  width: 90px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 28px;
+  color: #28352f;
+  margin: 10px 12px;
+  text-align: center;
```

Comment:
- [ ]  Instead of using `display: flex;` multiple times, kindly create a class say
``` css
.flex {
  display: flex;
}
```
and use the class in the HTML element that needs it. This will help your code to follow the DRY rule.